packages/osd-pm/src/commands/bootstrap.ts

```diff
@@ -105,9 +105,17 @@ export const BootstrapCommand: ICommand = {
     await parallelizeBatches(batchedProjects, async (project) => {
       const cache = caches.get(project);
       if (cache && !cache.valid) {
-        log.info(`[${project.name}] running [osd:bootstrap] script`);
-        cache.file.delete();
-        await project.runScriptStreaming('osd:bootstrap');
+        // Explicitly defined targets override any bootstrap scripts
```

Comment:
I don't quite see where this id documented in the README or explained by the logged output? It seems like the guidance implied is that packages should (preferably) provide build target configs and not provide a bootstrap script, but that we'll support the bootstrap script for backwards compatibility. But if they have both, I'd expect some sort of warning along the lines of "[osd:bootstrap] script ignored because build targets are defined".

Reply:
I like the idea of logging that. Lemme add it.